.gitignore

```diff
@@ -22,6 +20,8 @@ CMakeFiles
 /bin/
 /lib/
 config.h
+*.o
+*.obj
```

Comment:
Does cmake really put object files in the source tree?

Reply:
On my home machine cmake does put object files along sources, yes... maybe I set it up wrong? Or maybe different versions behave differently.